NodeJS/src/AgentClient.ts

```diff
@@ -9,27 +9,38 @@ export class AgentClient {
 
     private restClient: RestClient
     private subAccountId: string
+    private defaultSkillNumber: number
 
     constructor(subAccountId: string, restClient: RestClient) {
         this.restClient = restClient
         this.subAccountId = subAccountId
+        this.defaultSkillNumber = -1
+    }
+
+    getDefaultSkillNumber(){
```

Comment:
nit: I personally find [the `get` Accessor easier to read and use](https://www.typescriptlang.org/docs/handbook/classes.html#accessors). 

What you have works perfectly, I just wanted to point out Accessors in case you didn't know about them.

Reply:
Using accessor will force one to name the property differently, usually starts with _ and I do not like that.

Reply:
fair